src/app/containers/Navigation/index.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldShallowMatchSnapshot } from '../../../testHelpers';
+import NavigationContainer from './index';
+import igboConfig from '../../lib/config/services/igbo';
+
+jest.mock('react', () => {
+  const original = jest.requireActual('react');
+  return {
+    ...original,
+    useContext: jest.fn(),
+  };
+});
+
+const { useContext } = jest.requireMock('react');
+
+describe('Navigation Container', () => {
+  beforeEach(() => {
+    useContext.mockReturnValue(igboConfig);
+  });
+
+  afterEach(() => {
+    useContext.mockReset();
+  });
+
+  describe('snapshots', () => {
+    shouldShallowMatchSnapshot(
+      'should render a Navigation with igbo links correctly',
+      <NavigationContainer />,
+    );
+  });
+
+  describe('assertions', () => {
+    it('should render a Navigation with a Skip to content link, linking to #content', () => {
+      const { container } = render(<NavigationContainer />);
+
+      const skipLink = container.getElementsByTagName('a')[0];
```

Comment:
If we are interested on the first `a` only, then using `querySelector('a')` looks cleaner, and we don't have to access indexes.